submissions/Levi123/oop-exercise/index.js

```diff
@@ -0,0 +1,90 @@
+// class Animal {
+//    constructor (species, name, gender){
+//       this.species = species;
+//       this.name = name;
+//       this.gender = gender;
+//    }
+// }
```

Comment:
Don't keep comments in production code.
Exception: comments that explain reasons behind implementation peculiarities, that cannot be understood from the code itself (e.g. we cache data because server is down often and we want users to have some data anyways)